src/redux/usersReducer.js

```diff
@@ -29,7 +33,19 @@ const usersReducer = (state = initialState, action) => {
       };
 
     case ACTION_TYPES.SET_USERS: {
-      return { ...state, users: [...state.users, ...action.users] };
+      return { ...state, users: action.users };
```

Comment:
Consider storing "users" in "action.payload"

Reply:
done